build/Makefile

```diff
@@ -38,7 +38,11 @@ REGISTRY ?= gcr.io/agones-images
 # Where the kubectl configuration files are being stored
 KUBEPATH ?= ~/.kube
 # The (gcloud) test cluster that is being worked against
-CLUSTER_NAME ?= test-cluster
+GCP_CLUSTER_NAME ?= test-cluster
+GCP_CLUSTER_ZONE ?= us-west1-c
+GCP_CLUSTER_LEGACYABAC ?= false
+GCP_CLUSTER_NODEPOOL_INITIALNODECOUNT ?= 3
+GCP_CLUSTER_NODEPOOL_MACHINETYPE ?= n1-standard-4
```

Comment:
Should we move
```
GCP_CLUSTER_LEGACYABAC ?= false
GCP_CLUSTER_NODEPOOL_INITIALNODECOUNT ?= 3
GCP_CLUSTER_NODEPOOL_MACHINETYPE ?= n1-standard-4
```
To being target specific variables to `gcloud-test-cluster`, since they aren't reused across more than 1 target.

WDYT?

([here is an example](https://github.com/GoogleCloudPlatform/agones/blob/master/build/Makefile#L121), if you aren't familiar with them)

Reply:
Good idea, will do !